app/views/issues/show.html.erb

```diff
@@ -3,12 +3,25 @@
     <%= render 'issue_body', issue: @issue %>
   </div>
   <hr>
+  <h3><%= t('issues.issue_documents') %></h3>
   <div id="documents">
     <% @issue.documents.each do |document| %>
-      <%= link_to document.blob.filename, rails_service_blob_path(document.blob, filename: document.blob.filename) %>
+      <%= link_to document.blob.filename, url_for(document) %>
+      <% if can? :destroy, @issue %>
+        <%= link_to '<i class="fas fa-trash-alt"></i>'.html_safe, delete_document_attachment_project_issue_url(@project, @issue, document.id),method: :delete, data: { confirm: t('shared.deletion_confirmation_message', resource: t('issues.issue_document')) }, class: 'btn btn-sm btn-outline-danger card-button float-right' %>
```

Comment:
use link_to with block instead of passing icon here.

Reply:
done